lib/vault.js

```diff
@@ -0,0 +1,71 @@
+const { exec } = require('child_process');
```

Comment:
Add `"use strict";` to the top of the file to ensure we are in strict mode, it will help find some bugs.